ci-operator/step-registry/hypershift/aws/run-e2e/external/hypershift-aws-run-e2e-external-commands.sh

```diff
@@ -49,5 +49,6 @@ hack/ci-test-e2e.sh -test.v \
   --e2e.additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)" \
   --e2e.aws-endpoint-access=PublicAndPrivate \
   --e2e.external-dns-domain=service.ci.hypershift.devcluster.openshift.com \
+  "${EXTRA_ARGS}" \
```

Comment:
This seems like too generic (dangerous) a mechanism for jobs to pass in raw flags.  What about a `MULTIARCH_ENABLED` flag the job can set and we do the flag translation in here, similar to `REQUEST_SERVING_COMPONENT_PARAMS`?

@stevekuznetsov what do you think?

Reply:
Yeah, good call - the pattern used above would be a better option.